src/auth-service/controllers/request.js

```diff
@@ -66,9 +54,7 @@ const candidate = {
               message: value.message,
               candidate: value.data,
             });
-          }
-
-          if (value.success === false) {
+          } else if (value.success === false) {
```

Comment:
resolving a runtime error around getting about "`[Can't set headers after they are sent to the client](https://stackoverflow.com/questions/7042340/error-cant-set-headers-after-they-are-sent-to-the-client)`"

This error usually happens when you send several responses for one request, or for this case not doing proper if-else where appropriate.